project/AppDependencies.scala

```diff
@@ -2,20 +2,20 @@ import play.core.PlayVersion
 import sbt._
 
 object AppDependencies {
-  val bootStrapVersion = "7.12.0"
+  val bootStrapVersion = "7.19.0"
 
   lazy val dependencies = Seq(
     "uk.gov.hmrc"             %% "bootstrap-backend-play-28"      % bootStrapVersion
   )
 
   def testDependencies(scope: String) = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % bootStrapVersion    % scope,
-    "com.vladsch.flexmark"    %  "flexmark-all"                   % "0.36.8"            % scope,
-    "org.scalaj"              %% "scalaj-http"                    % "2.4.2"             % scope,
-    "com.github.tomakehurst"  %  "wiremock-jre8-standalone"       % "2.35.0"            % scope,
-    "org.mockito"             %% "mockito-scala-scalatest"        % "1.16.46"           % scope,
-    "io.cucumber"             %% "cucumber-scala"                 % "4.7.1"             % scope,
-    "io.cucumber"             %  "cucumber-junit"                 % "4.7.1"             % scope,
-    "com.novocode"            %  "junit-interface"                % "0.11"              % scope
-  )
+    "uk.gov.hmrc"             %% "bootstrap-test-play-28"         % bootStrapVersion,
+    "com.vladsch.flexmark"    %  "flexmark-all"                   % "0.36.8",
```

Comment:
I think this sort-of goes with scalatest. It could go to later/latest, e.g. 0.62.2.

Reply:
No the flexmark dep is a req for the bootstrap test lib
Latest one does not play well with this repo